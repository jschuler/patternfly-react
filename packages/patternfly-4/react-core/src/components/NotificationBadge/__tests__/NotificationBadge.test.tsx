// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { NotificationBadge } from '../NotificationBadge';
// any missing imports can usually be resolved by adding them here
import {  } from '..';

describe('NotificationBadge test', () => {
  it('NotificationBadge should match snapshot', () => {
    const view = shallow(
      <NotificationBadge
        isRead={false}
				children={<div>ReactNode</div>}
				className={"string"}
				aria-label={"string"}
      />);
    expect(view).toMatchSnapshot();
  });
});
