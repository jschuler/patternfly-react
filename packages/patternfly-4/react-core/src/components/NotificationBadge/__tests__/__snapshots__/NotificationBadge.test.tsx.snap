// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationBadge test NotificationBadge should match snapshot 1`] = `
<Component
  aria-label="string"
  className="string"
  variant="plain"
>
  <span
    className="pf-c-notification-badge pf-m-unread"
  >
    <div>
      ReactNode
    </div>
  </span>
</Component>
`;
